plugins/generator-1.18.2/forge-1.18.2/templates/rangeditem/rangeditem.java.ftl

```diff
@@ -137,7 +141,7 @@ public class ${name}Item extends Item {
 
 	<#if !data.ammoItem.isEmpty()>
 	if (entity.getAbilities().instabuild) {
-		entityarrow.pickup = AbstractArrow.Pickup.CREATIVE_ONLY;
+		entityarrow.pickup = AbstractArrow.Pickup.<#if !data.pickupProjectiles>CREATIVE_ONLY<#else>ALLOWED</#if>;

```

Comment:
This should remain `CREATIVE_ONLY`, or else players in survival can harvest projectiles shot in creative mode